org.gnome.gThumb.yml

```diff
@@ -0,0 +1,102 @@
+app-id: org.gnome.gThumb
+runtime: org.gnome.Platform
+runtime-version: '40'
+sdk: org.gnome.Sdk
+
+command: gthumb
+
+finish-args:
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --device=dri
+  - --share=ipc
+  - --talk-name=org.freedesktop.secrets
+  - --filesystem=xdg-cache
+  - --filesystem=xdg-config/gtk-3.0
```

Comment:
Excuse my laziness (I guess it was explained already in the old PR?), but what these two are needed for?